Libraries/PeriphDrivers/Source/GPIO/gpio_me30.c

```diff
@@ -100,165 +100,152 @@ int MXC_GPIO_Config(const mxc_gpio_cfg_t *cfg)
     }
 
     // Configure the pad
-    // TODO(ME30): "ps" (weak vs strong pull-up/down select) register field missing
     switch (cfg->pad) {
-    case MXC_GPIO_PAD_NONE:
-        gpio->padctrl0 &= ~cfg->mask;
-        gpio->padctrl1 &= ~cfg->mask;
-        break;
-
-    case MXC_GPIO_PAD_WEAK_PULL_UP:
-        gpio->padctrl0 |= cfg->mask;
-        gpio->padctrl1 &= ~cfg->mask;
-        // gpio->ps &= ~cfg->mask;
-        break;
-
-    case MXC_GPIO_PAD_PULL_UP:
-        gpio->padctrl0 |= cfg->mask;
-        gpio->padctrl1 &= ~cfg->mask;
-        // gpio->ps |= cfg->mask;
-        break;
-
-    case MXC_GPIO_PAD_WEAK_PULL_DOWN:
-        gpio->padctrl0 &= ~cfg->mask;
-        gpio->padctrl1 |= cfg->mask;
-        // gpio->ps &= ~cfg->mask;
-        break;
-
-    case MXC_GPIO_PAD_PULL_DOWN:
-        gpio->padctrl0 &= ~cfg->mask;
-        gpio->padctrl1 |= cfg->mask;
-        // gpio->ps |= cfg->mask;
-        break;
-
-    default:
-        return E_BAD_PARAM;
+        // Configure the pad
+        switch (cfg->pad) {
+        case MXC_GPIO_PAD_NONE:
+            gpio->padctrl0 &= ~cfg->mask;
+            break;
+
+        case MXC_GPIO_PAD_PULL_UP:
+            gpio->padctrl0 |= cfg->mask;
+            gpio->pssel |= cfg->mask;
+            break;
+
+        case MXC_GPIO_PAD_PULL_DOWN:
+            gpio->padctrl0 |= cfg->mask;
+            gpio->pssel &= ~cfg->mask;
+            break;
+
+        default:
+            return E_BAD_PARAM;
+        }
+
+        // Configure the drive strength
+        if (cfg->func == MXC_GPIO_FUNC_IN) {
+            return E_NO_ERROR;
+        } else {
+            return MXC_GPIO_SetDriveStrength(gpio, cfg->drvstr, cfg->mask);
+        }
     }
 
-    // Configure the drive strength
-    if (cfg->func == MXC_GPIO_FUNC_IN) {
-        return E_NO_ERROR;
-    } else {
-        return MXC_GPIO_SetDriveStrength(gpio, cfg->drvstr, cfg->mask);
+    /* ************************************************************************** */
+    uint32_t MXC_GPIO_InGet(mxc_gpio_regs_t * port, uint32_t mask)
+    {
+        return MXC_GPIO_RevA_InGet((mxc_gpio_reva_regs_t *)port, mask);
     }
-}
-
-/* ************************************************************************** */
-uint32_t MXC_GPIO_InGet(mxc_gpio_regs_t *port, uint32_t mask)
-{
-    return MXC_GPIO_RevA_InGet((mxc_gpio_reva_regs_t *)port, mask);
-}
 
-/* ************************************************************************** */
-void MXC_GPIO_OutSet(mxc_gpio_regs_t *port, uint32_t mask)
-{
-    MXC_GPIO_RevA_OutSet((mxc_gpio_reva_regs_t *)port, mask);
-}
+    /* ************************************************************************** */
+    void MXC_GPIO_OutSet(mxc_gpio_regs_t * port, uint32_t mask)
+    {
+        MXC_GPIO_RevA_OutSet((mxc_gpio_reva_regs_t *)port, mask);
+    }
```

Comment:
After line 136 to end of file includes additional tab, need to be removed.

Reply:
Done, thanks. 
note: it did looked weird to me too. i guess they're introduced after i ran 'auto correction' via clang-format. anyways, removed them manually this time